src/hooks.ts

```diff
@@ -143,7 +143,7 @@ function subscribeToInitialization(
             clientReady: false,
             didTimeout: false,
           });
-          res.dataReadyPromise!.then(() => {
+          res?.dataReadyPromise?.then(() => {
```

Comment:
can we please update it to`res.dataReadyPromise?.then()`?

res cannot be null or undefined as seen from the type, otherwise the condition inside the`if` at line 129 will throw before execution reaches this point. In this case res.dataReadyPromise is undefined.